nominees/opencollective.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Open Collective",
+  "description": "Open Collective is an online funding platform for open and transparent communities. We provide the tools to raise and share your finances in full transparency.",
+  "website": "https://opencollective.com/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/opencollective/documentation/blob/v2/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 17,
+      "evidenceURL": "https://opencollective.com/community"
+    }
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/opencollective",
```

Comment:
@ijeomaemeruwa could you update the repositoryURL to point to a specific repo instead of the Github organization for example [Open Collective](https://github.com/opencollective/opencollective). Thanks.